lib/js/components/Pagination/Pagination.vue

```diff
@@ -0,0 +1,369 @@
+<template>
+	<div
+		:class="{
+			pagination: true,
+			'-compact': layout === PAGINATION_LAYOUTS.COMPACT,
+			'-centered': isCentered,
+		}"
+	>
+		<div class="pagination__itemsWrapper">
+			<div class="pagination__items -default">
+				<template v-for="(n, index) in items">
+					<div :key="index" class="pagination__itemWrapper">
+						<a
```

Comment:
Does it make sense to use `a` when there is no URL? We have some global styles for anchors in wnl-platform that we'd have to be override:

![Screenshot 2022-11-25 at 14 57 55](https://user-images.githubusercontent.com/7030884/204000548-0188f504-b3ce-4ea8-a1de-b76376c7701e.png)